Modules/_io/bytesio.c

```diff
@@ -636,22 +636,18 @@ bytesio_iternext(bytesio *self)
 
 /*[clinic input]
 _io.BytesIO.seek
-    pos: Py_ssize_t
-    whence: int = 0
+    offset as pos: Py_ssize_t
+    whence: int(c_default='0') = io.SEEK_SET
     /
 
-Change stream position.
+Change stream position and return the new absolute position.
 
-Seek to byte offset pos relative to position indicated by whence:
-     0  Start of stream (the default).  pos should be >= 0;
-     1  Current position - pos may be negative;
-     2  End of stream - pos usually negative.
-Returns the new absolute position.
+Set the byte offset 'offset', relative to position indicated by 'whence':
```

Comment:
I think it would be better to keep a consistent terminology instead of alternating between "offset" and "position". "position" is probably more easily understood IMHO.

Reply:
Position is used to describe the _whence_ param. Would it not be confusing if 'position' is used to describe both parameters?

Reply:
Well the summary string does say "Change stream position and return the new absolute position". In the next line "stream position" suddenly becomes "byte offset".

Reply:
But this could also be "Set the stream position to the byte 'offset', relative to ...".

Reply:
```suggestion
Set the stream position to the byte offset 'offset', relative to the position indicated by 'whence':
```